### PR TITLE
Add AddPerformanceMaxForTravelGoalsCampaign example

### PR DESCRIPTION
**Pull request by jradcliff**
Based on https://github.com/googleads/google-ads-php/blob/main/examples/HotelAds/AddPerformanceMaxForTravelGoalsCampaign.php.

### CODE REVIEW

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java [hunk 1]
```diff
@@ -0,0 +1,957 @@
...
+                        "My Hotel propery asset set #" + CodeSampleHelper.getPrintableDateTime())
+                    .setType(AssetSetType.HOTEL_PROPERTY))
+            .build();
+    try (AssetSetServiceClient assetSetServiceClient =
+        googleAdsClient.getLatestVersion().createAssetSetServiceClient()) {
+      MutateAssetSetsResponse mutateAssetSetsResponse =
+          assetSetServiceClient.mutateAssetSets(
+              Long.toString(customerId), ImmutableList.of(assetSetOperation));
+      String assetSetResourceName = mutateAssetSetsResponse.getResults(0).getResourceName();
+      System.out.printf("Created an asset set with resource name: %s%n", assetSetResourceName);
```
[the remark this answers is not in the document]
Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java:427]: The `%n` is to add a newline char (`printf` won't do this automatically), so this looks correct to me, but please reopen if I missed something.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java [hunk 2]
```diff
@@ -0,0 +1,957 @@
...
+      HotelAssetSuggestion hotelAssetSuggestion) {
+
+    // Uses the GoogleAdService to create multiple text assets in a single request.
+    List<MutateOperation> mutateOperations = new ArrayList<>();
+
+    // First, adds all the text assets of the specified asset field type.
+    // Filters to only the specified asset field type.
+    // Constructs a mutate operation to create the asset.
+    // Adds the operation to the list.
+    for (HotelTextAsset asset : hotelAssetSuggestion.getTextAssetsList()) {
```
[the remark this answers is not in the document]
Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java]: Indeed I did! Thanks for catching that. Fixed.

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java [hunk 3]
```diff
@@ -0,0 +1,957 @@
...
+ *   <li>You must have at least one conversion action in the account. For more about conversion
+ *       actions, see
+ *       https://developers.google.com/google-ads/api/docs/conversions/overview#conversion_actions.
+ * </ul>
+ *
+ * <p>Notes:
+ *
+ * <ul>
+ *   <li>This example uses the default customer conversion goals. For an example of setting
+ *       campaign-specific conversion goals, see ShoppingAds/AddPerformanceMaxRetailCampaign.php.
```
[the remark this answers is not in the document]
Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java]: Done

File: google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (jradcliff) [reply google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddPerformanceMaxForTravelGoalsCampaign.java]: Ah, thanks. Fixed!

**Review by fiboknacky — APPROVED** [review]

**Review by AnashOommen — APPROVED** [review]

**Pull request merged**